scylla/templates/scylla-statefulset.yaml

```diff
@@ -20,6 +20,8 @@ spec:
         app: {{ template "scylla.name" . }} 
         release: {{ .Release.Name }}
     spec:
+      nodeSelector:
```

Comment:
Please use the standard way of doing this. e.g From https://github.com/src-d/charts/blob/master/borges/templates/consumer-deployment.yaml#L124-L126

```
    {{- if .Values.nodeSelector }}
      nodeSelector:
{{ toYaml .Values.nodeSelector | indent 8 }}
    {{- end }}
```